packages/knip/test/cli-reporter-markdown.test.ts

```diff
@@ -16,16 +16,16 @@ test('knip --reporter markdown', () => {
 
 ## Unlisted dependencies (2)
 
-| Name            | Location     |
-|:----------------|:-------------|
-| unresolved      | src/index.ts |
-| @org/unresolved | src/index.ts |
+| Name            | Location     | Severity |
+| :-------------- | :----------- | :------- |
+| @org/unresolved | src/index.ts | error    |
+| unresolved      | src/index.ts | error    |
 
 ## Unresolved imports (1)
 
-| Name         | Location     |
-|:-------------|:-------------|
-| ./unresolved | src/index.ts |
+| Name         | Location            | Severity |
+| :----------- | :------------------ | :------- |
+| ./unresolved | src/index.ts (8:23) | error    |
```

Comment:
TIL this format `src/index.ts (8:23)` can be copy-pasted and VS Code goes to that pos (I was used to only `src/index.ts:8:23`).

Reply:
Oh yeah, I will change it

Reply:
Well I'll leave it up to you, but I think your initial version is more readable (which is basically the purpose of Markdown reporter), and it's still easily copy-pastable.